courtroom.cpp

```diff
@@ -703,7 +703,7 @@ void Courtroom::enter_courtroom(int p_cid)
   QString char_path = ao_app->get_character_path(current_char);
 
   if (ao_app->custom_objection_enabled &&
-      file_exists(char_path + "custom.gif") &&
+      (file_exists(char_path + "custom.gif") || file_exists(char_path + "custom.apng")) &&
       file_exists(char_path + "custom.wav"))
```

Comment:
You might want to take a look at changing this line if you want to pave support for Opus interjections.

Reply:
Was going to do that in a follow up PR, to keep thing atomized.